/*
 * Copyright (C) Mellanox Technologies Ltd. 2001-2013.  ALL RIGHTS RESERVED.
 *
 * This software product is a proprietary product of Mellanox Technologies Ltd.
 * (the "Company") and all right, title, and interest in and to the software product,
 * including all associated intellectual property rights, are and shall
 * remain exclusively with the Company.
 *
 * This software is made available under either the GPL v2 license or a commercial license.
 * If you wish to obtain a commercial license, please contact Mellanox at support@mellanox.com.
 */

/*
* Each socket can write up to TCP_SND_BUF bytes (unsent+unacked) before blocking and waiting for acks.
* Increase the number to have better throughput, but do not increase it to much, otherwise latency performance will decrease.
* In the future, should be linked with setsockopt option.
*/

--- orig_lwip/arch/lwipopts.h	2013-03-24 15:55:48.088066000 +0200
+++ lwip/arch/lwipopts.h	2013-03-24 15:57:20.462574000 +0200
@@ -429,7 +429,7 @@
 #ifdef _LWIP_MIN_MEM_MODE
 #define TCP_SND_BUF                     4096 //256*1024
 #else
-#define TCP_SND_BUF                     40000 //256000
+#define TCP_SND_BUF                     100000 //256000
 #endif
 
 #define TCP_SND_BUF_NO_NAGLE 256000
